addons/event/static/src/client_action/event_barcode.scss

```diff
@@ -32,7 +60,10 @@
         -moz-border-radius: 10px;
         box-shadow: 2px 2px 10px rgba(0, 0, 0, 0.6);
         font-size: 1.2em;
-        padding: 3em;
+        padding-top: 0;
+        padding-left: 1em;
+        padding-right: 1em;
+        padding-bottom: 0.75em;
```

Comment:
we can put them all in the same padding attribute, right? 
padding: top right bottom left;
padding: 0 1em 1em .75em;